buildSrc/src/main/kotlin/packetevents.library-conventions.gradle.kts

```diff
@@ -30,6 +30,12 @@ tasks {
         options.release = 8
     }
 
+    processResources {
+        filesMatching(listOf("plugin.yml", "bungee.yml", "velocity-plugin.json", "fabric.mod.json")) {
```

Comment:
The inputs should be set here to make gradle not use the cache when just changing the version

Reply:
I am not entirely sure what you mean here? Could you elaborate?

Reply:
just so `inputs.property("version", project.version)` before this line

Reply:
Should be good now, right?